scripts/email.js

```diff
@@ -108,36 +102,60 @@ const EmailForm = {
       return;
     }
 
-
-    if (form.classList.contains("mieco-form")) {
-    postToEmailServer(
-        { ...params, message_id: "mieco" },
-        EmailForm.handleFormSuccess,
-        EmailForm.handleFormError
-    );
-    } else if (interests.includes("newsletter")) {
-        // This will only come from the innovation home page and will post to Basket
-      postToEmailServer(
-        {...params,
-            format: "H",
-            country: "us",
-            lang: "en",
-            newsletters: "mozilla-technology",
-            message_id: "innovations"
-        },
-        EmailForm.handleFormSuccess,
-        EmailForm.handleFormError
-      );
-    }
-    if (interests.includes("collaboration")) {
-      postToEmailServer(
-        { ...params,
-            website: website.value || "",
-            message_id: "innovations"
-        },
-        EmailForm.handleFormSuccess,
-        EmailForm.handleFormError
-      );
+    if (isBuilderPage) {
+      const newsletters = interests.length > 0 ? `mozilla-ai-challenge, ${interests}` : "mozilla-ai-challenge"
+      const params = { email, newsletters }
+      postToEmailServer(params, EmailForm.handleFormSuccess, EmailForm.handleFormError);
+    } else {
+      const name = form.querySelector('input[id="name"]').value;
+      const description = form.querySelector("textarea").value;
+      const website = form.querySelector('input["name=website"]');
```

Comment:
**existing bug**: noticed while testing branch and confirmed on main. The website selector errors on http://localhost:8000/mieco#newsletter-section.
<img width="505" alt="websites-input" src="https://user-images.githubusercontent.com/19650432/224677800-03a98c90-e156-4d18-b519-04f0532caae4.png">

**suggestion**: move the websites selector query inside `isInnovationPage` block?